code/datums/ruins/space.dm

```diff
@@ -350,3 +350,9 @@
 	suffix = "dangerous_research.dmm"
 	name = "Alternate Sciences Research Center"
 	description = "When you're messing with the occult, who knows what you're going to get?"
+
+/datum/map_template/ruin/space/travellersrest
```

Comment:
`travellers_rest`

Reply:
just in this one spot or for all instances of this word?

Reply:
All preferably

Reply:
All changed